monitor/monitoring-dam-generator-core/src/main/java/eu/seaclouds/monitor/monitoringdamgenerator/dcgenerators/MODACloudsDcGenerator.java

```diff
@@ -17,10 +15,10 @@
             .getLogger(MODACloudsDcGenerator.class);
 
     private static final String MODACLOUDS_DC_ID = "modacloudsDc";
-    private static final String START_SCRIPT_URL = "http://start.sh/";
+    private static final String START_SCRIPT_URL = "http://installMODACloudsDc.sh/";
```

Comment:
This URL will be updated as soon as this PR will be accepted since they need to point to the URL of the two scripts under the resources folder. So when this scripts will be updated somewhere (in our master according to this PR), those URLs need to be updated with a next PR. In any case they need to be URLs and not paths, I think @kiuby88 can be more precise on this last point.